workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/add_derived.py

```diff
@@ -72,14 +72,16 @@ def _column_pq2(ds: xr.Dataset) -> xr.DataArray:
 
 def _column_dq1(ds: xr.Dataset) -> xr.DataArray:
     if "net_heating_due_to_machine_learning" in ds:
-        column_dq1 = ds.net_heating_due_to_machine_learning
+        # fix isochoric vs isobaric transition issue
+        column_dq1 = 716.95 / 1004 * ds.net_heating_due_to_machine_learning
```

Comment:
Should this use the same physical constants as the prognostic run?

Also, what is the roadmap for removing support for the old names? It would be nice to at least raise a deprecation warning if the old names are encountered.

Reply:
These are the same constants used in VCM for the column heating functions: 1004 for cp and 287.05 for Rd and therefore 716.95 for cv. We might want to consider computing using a moist version of cp/cv, which is what the Fortran model does (e.g. for total energy) but that is a separate change IMO.

I added deprecation warnings.